model/plugins/create_user/structures.smithy

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+
+@input
+structure CreateUser_Input {
+    @required
+    @httpLabel
+    username: String
+
+    @required
+    @httpPayload
+    user: User
+}
+
+@output
+structure CreateUser_Output {
```

Comment:
For this API, the response is https://opensearch.org/docs/2.7/security/access-control/api/#example-response-8, while the `Response` structure contains a status and a message?

Reply:
> For this API, the response is https://opensearch.org/docs/2.7/security/access-control/api/#example-response-8, while the `Response` structure contains a status and a message?

Yes
![image](https://github.com/opensearch-project/opensearch-api-specification/assets/78027392/15cf4bbe-32ba-4069-bd74-93f7fba635d9)

Reply:
Oh my bad, this is the create user and not the get user.